tools/IceRpc.Slice.Tools/README.md

```diff
@@ -50,11 +50,10 @@ any of its subdirectories.
 The Slice directories are an aggregate of the `SliceDirectory` defined in your project (if any) and the `SliceDirectory`
 defined in NuGet packages referenced by your project.
 
-For example:
+For example, if your project's Slice files reference definitions from a common/slice directory:
```

Comment:
For example, if your project's Slice files reference definitions in directory `common/slice`: